rca/programmes/migrations/0062_add_subject_slugs.py

```diff
@@ -0,0 +1,21 @@
+from django.db import migrations, models
+
+from rca.programmes.models import Subject
```

Comment:
models should be imported using `apps.get_model()` https://docs.djangoproject.com/en/3.2/topics/migrations/#data-migrations